styles/components/_alert.scss

```diff
@@ -0,0 +1,21 @@
+.alert {
+  display: flex;
+  border-width: 2px;
+  border-style: solid;
+  border-color: none;
```

Comment:
Remove and the border change can be removed also

Reply:
done